src/PoolManager.sol

```diff
@@ -174,26 +178,38 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, Claims {
         if (pools[id].isNotInitialized()) revert PoolNotInitialized();
     }
 
+    /// @notice This will revert if a function is called by any address other than the current locker OR the most recently called, pre-permissioned hook.
     modifier onlyByLocker() {
```

Comment:
nit: now that `onlyByLocker` is on `initialize` I think `onlyByLocker` and `_checkLocker` should be higher in the contact